src/components/CallView/shared/VideoBackground.vue

```diff
@@ -72,10 +72,15 @@ export default {
 	},
 
 	async beforeMount() {
-		const response = await axios.get(generateUrl(`avatar/${this.user}/1`))
-		if (response.headers[`x-nc-iscustomavatar`] === '1') {
-			this.hasPicture = true
+		try {
+			const response = await axios.get(generateUrl(`avatar/${this.user}/1`))
```

Comment:
why `/1` ?
Why not get the full image directly for the cache?

Reply:
How should I do this? Can I use that image in the template instead of the url?

Reply:
Well reusing is one thing, but if you query /128 it might read the data from local cache?

Reply:
I'm just checking if it has a picture or not here, that's why I was trying to keep it as small as possible. But if I can use this image in the template somehow, It would make sense to have it bigger

Reply:
Yep, cache is working.
Just fetch directly the image you need.
(if you try with dev tools opened, it might fetch it twice because of the cache disabled option)